src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java

```diff
@@ -123,7 +127,8 @@ protected void preRun(SecurityAnalysisRunContext runContext) {
     }
 
     @Override
-    protected void postRun(SecurityAnalysisRunContext runContext) {
+    protected void postRun(SecurityAnalysisRunContext runContext, AtomicReference<ReportNode> rootReportNode, SecurityAnalysisResult ignoredResult) {
+        super.postRun(runContext, rootReportNode, ignoredResult);
```

Comment:
In my opinion, super call should be at the end bcs the below code enriches report

Reply:
Done.